src/main/java/tech/fastj/math/Maths.java

```diff
@@ -201,4 +201,15 @@ public static float denormalize(float normalizedNum, float min, float max) {
 
         return (normalizedNum * (max - min)) + min;
     }
+    
+    /**
+     * Returns a {@code float} based on the value between {@code startingVal} and {@code endingVal}.
+     * 
+     * @param startingVal Starting value.
+     * @param endingVal Ending value.
+     * @param value The value between the starting and ending values. 
+     **/
+    public static float inverseLerp(float startingVal, float endingVal, float value) {
```

Comment:
Might I suggest some names? You got pretty close overall, though I think you might benefit from these names:
`start, end, value`